TACC/tup-ui
# Task/TUP-357: my tickets dashboard component

## Overview:
Render tickets table to the dashboard

## Related:
- [TUP-357](https://jira.tacc.utexas.edu/browse/TUP-357)

## Changes:

## Testing:
1. Go to http://localhost:8000/dashboard
2. Ensure that the text data for your TACC tickets are correct in the new tickets table

## UI:
![Screenshot 2022-11-09 at 11 25 57 AM](https://user-images.githubusercontent.com/79928051/200899466-73899d1d-0304-4e7e-8313-b74f6a7c5850.png)


## Notes:
Main goal in this PR, per the Jira task, was to render a user's ticket data in a table in the dashboard. Therefore, the page that is rendered after clicking on a link in the table should be considered to be a placeholder. (I just wanted to make sure the links worked as intended.) Same thing if you click on the '+ New Ticket' button -- the modal that pops up is just a placeholder.
